components/StreamField/blocks/TopicsBlock/blocks.tsx

```diff
@@ -1,24 +1,26 @@
 import Image from 'next/image';
+import Link from 'next/link';
 
-import statusGreen from '../../../../public/images/status-green.svg';
-import statusAmber from '../../../../public/images/status-amber.svg';
-import statusRed from '../../../../public/images/status-red.svg';
+import { getTopicStatusImage } from '../../../../lib';
 
 import type { Topic } from './types';
 import styles from './TopicsBlock.module.scss';
 
-const statusImages = {
-    green: statusGreen,
-    red: statusRed,
-    amber: statusAmber,
-};
-
-export const TopicBlock = ({ title, statusColor, introduction }: Topic) => (
+export const TopicBlock = ({
+    meta,
+    title,
+    statusColor,
+    introduction,
+}: Topic) => (
     <div className={styles.card}>
         <div className={styles.card__status}>
-            <Image src={statusImages[statusColor]} alt="" />
+            <Image src={getTopicStatusImage(statusColor)} alt="" />
         </div>
-        <h3 className={styles.card__title}>{title}</h3>
+        <h3 className={styles.card__title}>
+            <Link href={`/${meta.slug}`}>
+                <a>{title}</a>
+            </Link>
```

Comment:
**Question**

Is enclosing `title` in `<a>` needed? Shouldn't the `Link` component render that `a` tag?